### DISCUSSION TITLE
Modify size and position of camera texture.

### DISCUSSION THREAD

**Discussion by ascoider** [asus4/tf-lite-unity-sample#284] [Q&A] [answered]
Hi! I keep working on the blaze sample. I was wondering which script should I modify to be able to change the size and the position of the webcam view, for example to project it half size in one side of the canvas and display something else like a background on the other. Thanks in advance.

**Comment by asus4** [comment, marked as answer]
Hi @ascoider 

In the BlazePose example, the webcam texture is rendered in fullscreen by the `CameraTextureBackground.cs` attached to the main camera. And the `WebCamInput` sets the texture via the `On Texture Updagte` event.

![Screenshot 2023-02-16 at 23 44 58](https://user-images.githubusercontent.com/357497/219505281-18cb651d-3869-4368-9337-26fa67750510.png)

You can replace it with another component, such as `RawImage`  👍 
![Screenshot 2023-02-16 at 23 48 26](https://user-images.githubusercontent.com/357497/219504789-316ff847-d4e1-4b23-9957-9d569e446ec1.jpeg)

**Comment by ascoider** [comment]
Thanks! Your advice works perfect and I also discovered I can modify the output field in the camera for the same purpose.
